src/test/java/org/opensearch/neuralsearch/common/BaseNeuralSearchIT.java

```diff
@@ -623,6 +626,28 @@ protected void deleteModel(String modelId) {
         );
     }
 
+    protected void pollForModelState(String modelId, Set<MLModelState> exitModelStates, int intervalMs, int maxAttempts)
```

Comment:
can we make interval and maxAttempts a class level constants, like DEFAULT_<item_name> and remove them from a method signature? That can be added later if needed, but most of the times I think default will just work.